src/ResourceManagement/RecoveryServices.Backup/RecoveryServices.Backup.Tests/TestHelpers/BackupDefinition.cs

```diff
@@ -15,6 +15,12 @@
 
 namespace Microsoft.Azure.Management.RecoveryServices.Backup.Tests
 {
+    /// <summary>
+    /// For the recovery services backup tests, need some test artifact to be created manually for the tests to run:
```

Comment:
I've moved the test artifact related values to a json file and have included this detail in the comment.